src/src/pages/Dashboard/components/QuickStart/QuickStart.tsx

```diff
@@ -28,24 +31,22 @@ function QuickStart() {
           tint={100}
           className='QuickStart__section__title'
         >
-          Integrate Aim with your code
+          Integrate FasttrackML with your code
         </Text>
         <CodeBlock
-          code={`from aim import Run
+          code={`from mlflow import log_metric, log_param
 
-# Initialize a new run
-run = Run()
+# Set FasttrackML tracking server
+mlflow.set_tracking_uri("${fasttrack_server}")
 
-# Log run parameters
-run["hparams"] = {
-    "learning_rate": 0.001,
-    "batch_size": 32
-}
+# Log parameters
+log_param("learning_rate", 0.001)
+log_param("batch_size", 32)
 
 # Log metrics
 for i in range(10):
-    run.track(i, name='loss', step=i, context={ "subset":"train" })
-    run.track(i, name='acc', step=i, context={ "subset":"train" })`}
+    log_metric(key="loss", value=0.0 + i, step=i)
+    log_metric(key="acc", value=0.0 + i, step=i)`}
```

Comment:
I think Python should be able to convert `i` to a float automatically (which is cursed, I know...)